products/compute/terraform.yaml

```diff
@@ -1084,6 +1084,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           udp500_forwarding_rule_name: "fr-udp500"
           udp4500_forwarding_rule_name: "fr-udp4500"
           route_name: "route1"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "vpn_tunnel_beta"
```

Comment:
I think this file needs to be `git add`-ed

Reply:
gdi